packages/itwinui-react/src/core/ThemeProvider/ThemeProvider.tsx

```diff
@@ -61,9 +60,22 @@ type ThemeProviderOwnProps = Pick<RootProps, 'theme'> &
   (
     | {
         themeOptions?: RootProps['themeOptions'];
+        styleOptions?: {
+          /**
+           * If true, all styles will be wrapped in a cascade layer named `itwinui`.
+           * This helps avoid specificity battles with application styles.
+           *
+           * @default true
+           */
+          withLayer?: boolean;
```

Comment:
@gretanausedaite @LostABike Do you have a better naming suggestion?

I was thinking of renaming `styleOptions` to `autoImportStyles` and also allowing boolean. The boolean would be useful if somebody wants to import `@itwin/itwinui-css` manually.

```jsx
<ThemeProvider autoImportStyles={{ withLayer: false }}>
```
```jsx
<ThemeProvider autoImportStyles={false}>
```

Alternatively, keep `styleOptions` and rename `withLayer` to just `layer` or `layers`. Can also add `disabled` property if manually importing.
```jsx
<ThemeProvider styleOptions={{ layer: false }}>
```
```jsx
<ThemeProvider styleOptions={{ disabled: true }}>
```

Reply:
I feel `styleOptions` -> `styleImports` is a bit more concise than `autoImportStyles`. Allowing boolean would be nice.

Also I think this
```
<ThemeProvider autoImportStyles={false}>
```
is more intuitive than 
```
<ThemeProvider styleOptions={{ disabled: true }}>
```
at least for me.

Reply:
another option: `includeCss`

```jsx
<ThemeProvider includeCss={{ withLayer: false }}>
```
```jsx
<ThemeProvider includeCss={false}>
```

Reply:
> The boolean would be useful if somebody wants to import @itwin/itwinui-css manually.

Why would they want it?

> autoImportStyles={false}

Users might think that `autoImportStyles={false}` will not import styles at all and not just disables layers.

I like `styleImports` with options like `layers` (or `useLayers`) and `disabled` to disable all style imports.

Reply:
> > The boolean would be useful if somebody wants to import @itwin/itwinui-css manually.
> 
> Why would they want it?

it can be useful if they want more control over the css. For example, inlining the styles into html or running postcss transformations.

> > autoImportStyles={false}
> 
> Users might think that `autoImportStyles={false}` will not import styles at all and not just disables layers.

yes that's what it does. it will not include styles at all.

> I like `styleImports` with options like `layers` (or `useLayers`) and `disabled` to disable all style imports.

`use` prefix is meant for hooks and i would like to avoid naming properties with this prefix.

Reply:
> <ThemeProvider includeCss={{ withLayer: false }}>

I like this one then ^

Reply:
^ agree with @gretanausedaite, wording for both is pretty clear in that case

Reply:
Renamed!